graphql-server/go-server/pkg/common/common.go

```diff
@@ -77,3 +77,10 @@ func SystemDatasourceOSS(ctx context.Context, mgrClient client.Client, dynamicCl
 	}
 	return datasource.NewOSS(ctx, mgrClient, dynamicClient, endpoint)
 }
+
+func TryToSetDeleting(status *string, obj *unstructured.Unstructured) {
```

Comment:
it's NOT a good name 'TryToSetDeleting', it should be something like 'GetObjStatus(obj, status)', and we may add more status later based on the data of obj.